extensions/guacamole-auth-restrict/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+
+    "guacamoleVersion" : "1.5.0",
```

Comment:
This should match the version of guacamole-ext that the extension was built against.

Reply:
Ah, yeah, need to go through and update everything to 1.5.2...

Reply:
Fixed via rebase and force-push.